src/Pages/Home.tsx

```diff
@@ -56,27 +82,44 @@ const Home: React.FC = () => {
       <section>
         <Card
           size="small"
-          title="About Us"
+          title={t('about.title')}
           style={{ width: '90vw', margin: 'auto' }}
         >
-          <h3>Our vision and mission</h3>
+          <h3>{t('about.visionMissionTitle')}</h3>
           <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut rhoncus
-            interdum ligula, sit amet ullamcorper sapien ullamcorper ut. Vivamus
-            dapibus, urna nec tempor luctus, mauris risus efficitur justo, et
-            gravida metus ex id enim.
+            {t('about.visionMissionText')
+              .split('\n')
+              .map((line: string, i: number) => (
+                <span key={i}>
+                  {line}
+                  <br />
+                </span>
+              ))}
           </p>
-          <h3>Who we are and where we stand</h3>
+          <h3>{t('about.whoWeAreTitle')}</h3>
           <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut rhoncus
-            interdum ligula, sit amet ullamcorper sapien ullamcorper ut. Vivamus
-            dapibus, urna nec tempor luctus, mauris risus efficitur justo, et
-            gravida metus ex id enim.
+            {t('about.whoWeAreText')
+              .split('\n')
+              .map((line: string, i: number) => (
+                <span key={i}>
+                  {line}
+                  <br />
+                </span>
+              ))}
           </p>
+
+          {/* Team members */}
+
+          <Row gutter={[16, 16]} justify="center">
```

Comment:
This doesn't seem to adjust dynamically for mobile layout, but we can look at that in a next iteration. Maybe something for @emgerold ?